src/components/seeds/database/Filters.tsx

```diff
@@ -127,19 +127,22 @@ export default function Filters(props: Props): JSX.Element {
           onClose={onClosePopover}
         />
         {props.columns.map((col) => {
-          return (
-            <div key={col.key} className={classes.pill}>
-              <Chip
-                id={`filter-${col.key}`}
-                variant='outlined'
-                size='medium'
-                label={getLabel(col)}
-                onClick={(event) => handleClick(event, col)}
-                icon={<ArrowDropDown />}
-                className={props.filters[col.key] ? classes.selectedFilter : ''}
-              />
-            </div>
-          );
+          if (col.filter?.type !== 'hidden') {
+            return (
+              <div key={col.key} className={classes.pill}>
+                <Chip
```

Comment:
we should move a chip component to web-components so designers can control it

Reply:
this is for later

Reply:
This chip is old, I think at some point we will remove them and use filters as in species page. We have a pill component there that should be moved into web-components

Reply:
Got it, the sooner we do this the better I think - so the visual updates can be controlled by design. 
Since we are working on this now.. ??? Or at least a fast follow up?

Reply:
I understood we were not going to update this filters UI now

Reply:
We don't need to change the filters.. just need to export this from web-components for designer control. So they can update styles on their own. If you think it's ok to keep it here that's fine too.

Reply:
What if move this to web-components we will need to create a **Chip** component that is going to be replaced by the **Pill** component later.

Reply:
If we have a concrete plan to replace it with Pill before the visual updates effort then maybe not worth it.
Otherwise, there are benefits to extracting this out - even if only for a few months. Again, we can keep it here as-is if there's a strong preference. Sometimes these plans get delayed and we will be stuck with this Chip.
Fwiw, the pill is not in web-components either. I created a ticket for this.

Reply:
Maybe best to get @chudo76 's thoughts on this :) I will ask in the Jira.

Reply:
Thanks @karthikbtf !